client/repl/client.go

```diff
@@ -1063,39 +1063,43 @@ func RunClient(serverHost string, serverPort int) error {
 
 	fmt.Printf(
 		"%s\n\n%s\n\n%s\n\n",
-		color.Aurora.Blue(strings.Trim(aldaASCIILogo, "\n")),
-		color.Aurora.Cyan(strings.Trim(aldaVersionText(serverVersion), "\n")),
-		color.Aurora.Bold("Type :help for a list of available commands."),
+		GookitColor.Blue.Render(strings.Trim(aldaASCIILogo, "\n")),
+		GookitColor.Cyan.Render(strings.Trim(aldaVersionText(serverVersion), "\n")),
+		GookitColor.Bold.Render("Type :help for a list of available commands."),
 	)
 
 	err = os.MkdirAll(filepath.Dir(replHistoryFilepath), os.ModePerm)
 	if err != nil {
 		return err
 	}
 
-	console, err := readline.NewEx(&readline.Config{
-		Prompt:          "alda> ",
-		InterruptPrompt: "^C",
-		EOFPrompt:       "^D",
-		HistoryFile:     replHistoryFilepath,
-	})
-	if err != nil {
-		return err
-	}
+	// temp File to store history for reverseSearch, up down previous commands, etc.
+	var history_fn = filepath.Join(os.TempDir(), ".alda_history")
```

Comment:
Let's use `replHistoryFilepath` here instead. On my system, that ends up being `~/.cache/alda/history/alda-repl-history`. (The location varies depending on OS)